misago/attachments/tests/test_store_uploaded_file.py

```diff
@@ -195,3 +195,24 @@ def test_store_uploaded_file_raises_validation_error_for_invalid_image(
 
     assert exc_info.value.message == "Image file is not valid."
     assert exc_info.value.code == "unidentified_image"
+
+
+@override_dynamic_settings(
+    attachment_image_max_width=400,
+    attachment_image_max_height=300,
+    attachment_thumbnail_width=150,
+    attachment_thumbnail_height=100,
+)
+def test_store_uploaded_file_stores_image_file_scaled_down_with_thumbnail(
```

Comment:
Please move this test under `test_store_uploaded_file_stores_image_file_scaled_down` so they are together